src/Containers/CadView.jsx

```diff
@@ -151,12 +151,12 @@ export default function CadView({
     const preselectMat = new MeshLambertMaterial({
       transparent: true,
       opacity: 0.5,
-      color: theme.palette.highlight.light,
+      color: theme.palette.highlight.main,
       depthTest: true,
     })
     const selectMat = new MeshLambertMaterial({
       transparent: true,
-      color: theme.palette.highlight.main,
+      color: theme.palette.secondary.main,
```

Comment:
this breaks high - lighting
<img width="737" alt="image" src="https://user-images.githubusercontent.com/3433606/201073400-428e6118-cb04-4230-9a73-5d878fa79d2c.png">

Reply:
So the highlighting that is active in production was broken but in a different way.  The console log shows the highlight color is undefined.. it was being passed in from our theme, but the theme property wasn't defined.  So, I picked what looked good to me last night :) but yeah, open to any suggestions for alternatives

Reply:
![image](https://user-images.githubusercontent.com/3433606/201207018-4da7af35-d14e-4c22-82fb-fbd718f3c155.png)